src/components/number-input/number-input.st.css

```diff
@@ -0,0 +1,73 @@
+:import {
+    -sb-from: "../../style/default-theme/variables.st.css";
+    -sb-named: color_MainText,
+               color_MainText_Background,
+               borderWidth,
+               inputHeight;
+}
+
+.number-input {
+    position: relative;
+    display: inline-block;
+}
+
+.stepper {
+    position: relative;
+
+    width: value(inputHeight);
+    height: value(inputHeight);
+}
+
+.stepper-control {
+    width: 100%;
+
+    display: flex;
+
+    justify-content: center;
+    align-content: center;
+    align-items: center;
+
+    position: absolute;
+
+    margin: 0;
+    padding: 0;
+
+    background-color: value(color_MainText_Background);
+    color: value(color_MainText);
+}
+
+.stepper-control-icon {
+    color: value(color_MainText);
+    width: 16px;
+    height: 16px;
+}
+
+.stepper-control:hover .stepper-control-icon,
+.stepper-control:focus .stepper-control-icon,
+.stepper-control:disabled .stepper-control-icon {
```

Comment:
will this work without explicit `.stepper-control-icon` in selector? Svg should use the same `currentColor`